packages/pyright-internal/src/languageService/completionProvider.ts

```diff
@@ -2669,10 +2674,20 @@ export class CompletionProvider {
                                         } else if (isOverloadedFunction(functionType)) {
                                             // 35 is completion tooltip's default width size
                                             typeDetail = getOverloadedFunctionTooltip(
```

Comment:
we should probably truncate all the additional overloads when asking for a completion. lots of extra work being done.